framework/assets/yii.gridView.js

```diff
@@ -123,7 +123,7 @@
             var $form = $('<form/>', {
                 action: url,
                 method: 'get',
-                class: 'gridview-filter-form',
+                'class': 'gridview-filter-form',
```

Comment:
@klimov-paul I think that quoting only `class` is enough, there is no point for quoting all keys.